openvdb/points/AttributeArray.h

```diff
@@ -127,7 +127,7 @@ class OPENVDB_API AttributeArray
     enum Flag {
         TRANSIENT = 0x1,            /// by default not written to disk
         HIDDEN = 0x2,               /// hidden from UIs or iterators
-        OUTOFCORE = 0x4,            /// data not yet loaded from disk
+        OUTOFCORE = 0x4,            /// data not yet loaded from disk (deprecated flag as of ABI=5)
```

Comment:
Is there any danger of completely removing this for ABI5?

Reply:
Only reason I didn't remove it is so that the same slot doesn't get accidentally reused to mean something different. It's marked deprecated in the comment, so I figured that was sufficient.